api/src/main/java/com/codeforcommunity/dto/site/GetSiteResponse.java

```diff
@@ -33,6 +35,7 @@ public GetSiteResponse(
     this.address = address;
     this.entries = entries;
     this.neighborhoodId = neighborhoodId;
+    this.plantingDate = plantingDate;
```

Comment:
🔧 You're writing exactly the same code twice. We purposefully made these Plain Old Java Objects so that we can abstract freely. Take common code and extract them into an abstract class ala FII.